pkg/websocket/handler.go

```diff
@@ -59,49 +63,63 @@ func (h *handler) Handle(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	h.conn = conn
+	listener := fmt.Sprintf("%s", uuid.NewV4())
```

Comment:
Maybe you should call the listener with a more explicit name. This way, if one day we wanna dump the whole map, we will see clearly which listener is associated to what.

For example : `fmt.Sprintf("ws_client_%s", uuid.NewV4())`

Reply:
:+1: